compiler/src/dotty/tools/dotc/transform/sjs/PrepJSInterop.scala

```diff
@@ -814,20 +822,41 @@ class PrepJSInterop extends MacroTransform with IdentityDenotTransformer { thisP
       super.transform(tree)
     }
 
+    /** Removes annotations from exported definitions (e.g. `export foo.bar`):
+     *  - `js.native`
+     *  - `js.annotation.*`
+     *  - `js.annotation.internal.*`
```

Comment:
When `stripJSAnnotsOnExported` is called, we've just checked that the symbol doesn't have any annotation from `internal.*`. So I believe removing stuff from `internal.*` is unnecessary.

Reply:
thank you for the explaination, I have removed the check for `internal.*`